page/docs/guide/blocks.md

```diff
@@ -6,6 +6,13 @@ sidebar_position: 8
 
 Devnet starts with a genesis block (with block number equal to `0` and block hash equal to `"0x0"`).
 
+Genesis block contains transactions for:
```

Comment:
I'm not sure if it's good to mention all of these contracts by name here. We already mention them in other places in the documentation (we don't mention chargeable account but that's because it's a development hack, not a user feature), but mentioning them here just introduces one more point of code/docs which needs to be modified.

Reply:
Didn't know that. Should I remove it?

Reply:
IMO you could.

Reply:
fixed in 7aa26ab4a246eb0ac6f66ac37fca08d7c38966df